manuscript/chapter1.md

```diff
@@ -359,59 +344,55 @@ let helloWorld = 'Welcome to the Road to learn React';
 helloWorld = 'Bye Bye React';
 ~~~~~~~~
 
-You would declare variables with `let` if you need to re-assign the variable later on.
+**TIP:** Declare variables with `let` if you think you'll want to re-assign it later on.
 
-However, you have to be careful with `const`. A variable declared with `const` cannot be modified. However, when the variable is an array or object, the value it holds can get updated. The value it holds is not immutable.
+Note that a variable declared directly with `const` cannot be modified. However, when the variable is an array or object, the values it holds can get updated through indirect means:
 
 {title="Code Playground",lang="javascript"}
 ~~~~~~~~
 // allowed
 const helloWorld = {
-  text: 'Welcome to the Road to learn React'
+  text: text: 'Welcome to the Road to learn React'
 };
 helloWorld.text = 'Bye Bye React';
 ~~~~~~~~
 
-When do you use each declaration? There are different opinions about the usage. I suggest using `const` whenever you can. It indicates that you want to keep your data structure immutable even though values in objects and arrays can be modified. If you want to modify your variable, you can use `let`.
+There are varying opinions about when to use *const* and when to use *let*. I would recommend using `const` whenever possible to keep your data structures immutable, so that you need only worry about the values contained in objects and arrays. Immutability is embraced in the React ecosystem, so `const` should be your default choice when you define a variable, though it’s not just about immutability, but about assigning variables only once. It shows the intent of not changing (re-assigning) the variable.
```

Comment:
"... so that you need only worry about the values contained in objects and arrays (hence const doesn't make your variable really immutable, because its content can still be changed). Immutability is embraced in the React ..."

Reply:
The sentence you want to add in parenthesis isn't necessary because that point was already made clear in the 2 sentences before.  It just makes the paragraph harder to read, because the reader will wonder why the sentence in parenthesis isn't just the next sentence.  
Also, we don't start sentences with "hence" or "therefore".  They are too fancy for simplifiued technical writing, and they always go in the middle to two dependent clauses separated by a semicolon.  Example: "It is raining; therefore, I will take an umbrella."  That sounds wordy, so we replace the semicolon(;) and "therefore" like this:  "It is raining, so I will take an umbrella."  Both say the same thing, except the second one doesn't have an unnecessary pause in the middle.